formasaurus/formtype_features.py

```diff
@@ -39,27 +40,24 @@ class FormElements(BaseFormFeatureExtractor):
     Features based on form HTML elements: counts of elements
     of different types, GET/POST form method.
     """
+
     def get_form_features(self, form):
         typecounts = _get_type_counts(form)
         return {
-            'has <textarea>': typecounts['textarea'] > 0,
-            'has <input type=radio>': typecounts['radio'] > 0,
-            'has <select>': typecounts['select'] > 0,
-            'has <input type=checkbox>': typecounts['checkbox'] > 0,
-            'has <input type=email>': typecounts['email'] > 0,
-
-            '2 or 3 inputs': len(form.inputs.keys()) in {2, 3},
```

Comment:
quite minor but would be great to preserve the grouping, e.g. with adding empty comments between the lines

Reply:
Done. Although I left the comments empty.

Reply:
Great, that's what I'd do as well